site/docs/relations/physical_relations.md

```diff
@@ -47,7 +47,45 @@ The nested loop join operator does a join by holding the entire right input and
+| Right Input               | A relational input.                                                                                                                                                                                                     | Required                 |
+| Left Keys                 | References to the fields to join on in the left input.                                                                                                                                                                  | Required                 |
+| Right Keys                | References to the fields to join on in the right input.                                                                                                                                                                 | Required                 |    
+| Post Join Predicate       | An additional expression that can be used to reduce the output of the join operation post the equality condition. Minimizes the overhead of secondary join conditions that cannot be evaluated using the equijoin keys. | Optional, defaults true. |
+| Join Type                 | One of the join types defined in the Join operator.                                                                                                                                                                     | Required                 |
+| Duplicate Eliminated Side | The side that is deduplicated and pushed into the other side.                                                                                                                                                           | Required                 |
+
+## Duplicate Eliminated Get Operator
+An operator that takes as its input the result of the deduplicated side of the Duplicate Eliminated Join. It simply scans the input and outputs the deduplicated values.
+| Signature            | Value                                                                               |
+| -------------------- |-------------------------------------------------------------------------------------|
+| Inputs               | 1                                                                                   |
+| Outputs              | 1                                                                                   |
+| Property Maintenance | Distribution is not maintained due to the deduplication. Orderedness is eliminated. |
+| Direct Output Order  | It will only project the deduplicated columns from it's input                       |
+
+### Duplicate Eliminated Get Properties
+
+| Property   | Description                                        | Required                    |
+|------------|----------------------------------------------------|-----------------------------|
+| Input      | A relational input.                                | Required                    |
+| Column IDs | The columns that were deduplicated from the input. | Required                    |
```

Comment:
```suggestion
```

Why do we need to specify this twice (here and in the duplicate eliminated join)?  This suggestion removes the property here in favor of the property on the duplicate eliminated join.  However, if you'd rather it be here (and remove the property from the duplicate eliminated join) then I think that's fine too.